ocfweb/docs/docs/docs/operatingrules/bylaws.md

```diff
@@ -114,12 +118,15 @@ amended at the Board meeting of December 2, 2019.
        1. The University Affairs Committee is headed by the General Managers,
           and an election for University Affairs Committee head will not be held.
 
-       2. Election dates for heads to the DeCal Committee and ad-hoc
+       2. The Service Committee is headed by the Site Managers, and an election
+          for Service Committee head will not be held.
+
+       3. Election dates for heads to the DeCal Committee and ad-hoc
           committees may be set by the OCF Decision Making Process. Election
           dates may not be set for fewer than 14 days in the future. Elections
           must be announced to all Board members at least 14 days in advance.
 
-       3. If there is no elected head for the Lab Space Committee, it is headed
+       4. If there is no elected head for the Lab Space Committee, it is headed
           by the Site Managers.
 
     4. The terms of permanent committee heads begin on the first day following
```

Comment:
I think this should be numbered 5

Reply:
I counted 4, hmm

Reply:
Oh my bad, my phone didn't show the indentation right, weird.